bumble/gatt.py

```diff
@@ -151,6 +151,14 @@
 # Battery Service
 GATT_BATTERY_LEVEL_CHARACTERISTIC = UUID.from_16_bits(0x2A19, 'Battery Level')
 
+# ASHA Service
+ASHA_SERVICE                             = UUID.from_16_bits(0xFDF0, 'Audio Streaming for Hearing Aid')
```

Comment:
those should be prefixed with `GATT_` like the other contants in this file.

Reply:
fixed